include/asterisk/acl.h

```diff
@@ -285,6 +285,32 @@ int ast_get_ip(struct ast_sockaddr *addr, const char *hostname);
  * \retval 0 Success
  * \retval -1 Failure
  */
+
+int ast_get_ip_or_srv_with_preference(struct ast_sockaddr *addr, const char *hostname,
+									  const char *service, struct ast_sockaddr *preference);
+/*!
```

Comment:
The documentation should go with the function it's for (...preference)

Reply:
Fixed (don't copy, paste, and make the updates in the wrong place)